tools/downloader/requirements-pytorch.in

```diff
@@ -1,4 +1,5 @@
 onnx
+packaging
```

Comment:
Why is this needed? Couldn't it be downloaded as a wheel?

Reply:
huggingface transformers use it for versions verfication, without it a lot of code required to be modified, ok, I will try to use it as wheel

Reply:
changed to download as wheel